### DISCUSSION TITLE
Using a copy of Camera Animation Blueprint

### DISCUSSION THREAD

**Discussion by woodencase01** [Sixze/ALS-Refactored#306] [Q&A]
I'm trying to customize the camera distance if we are aiming using one hand pistol, or two hand pistol, and I feel like it should be done within the animation blueprint.

After copying the original AB from the plugin and renaming it, I've changed it in the class default from my character.

When playing the game, most of the camera features aren't working: switching shoulders, aiming, etc.

Is there something I've been missing?

I prefer duplicating the AB as I can compare them later if there is an update with ALS-Refactored.

**Comment by MagentaPuppet** [comment]
You could duplicate and change the original instead of the duplicate, so, you'll still have the duplicate as a ref. It probably has some dependencies that you are not aware of. You can try using the reference viewer to see them.

**Reply by woodencase01** [reply]
Thanks for your suggestion, this is what I ended up doing. 

The reference viewer only points to the character blueprint, where the camera AB is specified in the class default. There are no other reference.

I agree with you, there must be something "hard coded" to the original AB.
